### PR TITLE
protontricks: add launch command

### PR DESCRIPTION
**Pull request by Managor**
- [ ] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [ ] The page description(s) have links to documentation or a homepage.
- [ ] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Review by gutjuri — APPROVED** [review]
Thanks!

File: pages/linux/protontricks.md [hunk 1]
```diff
@@ -23,6 +23,10 @@
 
 `protontricks -s {{game_name}}`
 
+- Run an executable in the proton environment of a specific game:
+
+`protontricks-launch --appid {{game_appid}} {{path/to/executable.exe}}`
```
Comment (spageektti) [inline pages/linux/protontricks.md:28]:
```suggestion
`protontricks-launch --appid {{appid}} {{path/to/executable.exe}}`
```
like in other examples

**Review by spageektti — COMMENTED** [review]
Thanks for your contribution

File: pages/linux/protontricks.md (hunk 1 above)
Comment (Managor) [reply pages/linux/protontricks.md:28]: I seems to have had a brainfart. Started typing "game_id", looked over to the other examples and wrote "game_appid" in the end.

**Review by spageektti — APPROVED** [review]

**Pull request merged**